doc/user_manual/postprocessor.tex

```diff
@@ -114,9 +114,15 @@ \subsubsection{BasicStatistics}
     \item \textbf{VarianceDependentSensitivity}: matrix of sensitivity coefficients dependent on the variance of the variables
     \item \textbf{samples}: the number of samples in the data set used to determine the statistics.
   \end{itemize}
-  If all the quantities need to be computed, this can be done through the \xmlNode{all} node, which
-  requires the \xmlNode{targets} and \xmlNode{features} sub-nodes.
-
+  This XML node needs to contain the attribute:
+  \begin{itemize}
+    \itemsep0em
+    \item \xmlAttr{prefix}, \xmlDesc{required string attribute}, user-defined prefix for the given \textbf{metric}.
+      For scalar quantifies, RAVEN will define a variable with name defined as:  ``prefix'' + ``\_'' + ``parameter anme''.
+      For matrix quantities, RAVEN will define a variable with name defined as: ``prefix'' + ``\_'' + ``target parameter name'' + ``\_'' +``feature parameter name''. 
+      This variable will be used by RAVEN for the internally calculations. It is also accessible by the user through
```

Comment:
"for the internally calculations": either "for internal calculations" or "internally" is fine

Reply:
fixed